docs/ARCHITECTURE.md

```diff
@@ -7,18 +7,12 @@ It is in charge of actions such as:
   * Creating new panes
   * Closing existing panes (and filling up their space by other panes)
 
-## Terminal Pane (zellij-server/src/pane/terminal_pane.rs)
+## Terminal Pane (zellij-server/src/panes/terminal_pane.rs)
 The TerminalPane represents a pane on screen that connects to a single pty (pseudo terminal) and (presumably) has one shell or other program running inside it.
 The TerminalPane has two main roles:
   * Keeping track of the Scroll, which represents the buffer of lines in this terminal
   * Interpreting the ANSI/VT instructions coming from the pty in order to adjust the styling of characters, change the cursor position, etc.
 
-### Scroll (src/terminal_pane/scroll.rs)
```

Comment:
Scroll functions seem to have been moved to `zellij-server/src/panes/terminal_pane.rs` in the refactoring process.

So I think we just need to modify the path.

Reply:
Thanks for the comment